packages/mux-uploader/package.json

```diff
@@ -42,6 +42,7 @@
     "build": "npm-run-all --parallel build:types 'build:esm -- --minify' 'build:iife -- --minify' 'build:cjs -- --minify' 'build:esm-module -- --minify'",
     "prebuild": "yarn clean",
     "prepublishOnly": "yarn build",
+    "deploy:canary": "npm version $(semver $(npm view @mux-elements/mux-uploader versions --json | jq -r '. - map(select(contains(\"alpha\") or contains(\"beta\"))) | last' | sed -r 's/-[a-z0-9]{7}$//g') -i prerelease --preid canary)-$(git rev-parse --short HEAD) --git-tag-version false && npm publish --tag canary --access public",
```

Comment:
this would likely fail until any version of `mux-uploader` is published to npm, since we ask npm for the latest version in this command (`npm view @mux-elements/mux-uploader versions --json`)

Reply:
cc @luwes